drivers/storage/ebs/tests/Vagrantfile

```diff
@@ -0,0 +1,487 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+require 'fileutils'
+require 'shellwords'
+require 'fog'
```

Comment:
Is fog included by default? Or is it a test dependency that needs to be installed?